packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts

```diff
@@ -13,13 +14,17 @@ export interface IBasePickerProps<T> extends React.Props<any> {
   /**
    * Function that specifies how an individual suggestion item will appear.
    */
-  onRenderSuggestionsItem?: (props: T) => JSX.Element;
+  onRenderSuggestionsItem?: (props: T, itemProps?: T) => JSX.Element;
   /**
    * A callback for what should happen when a person types text into the input.
    * Returns the already selected items so the resolver can filter them out.
    */
   onResolveSuggestions: (filter: string, selectedItems?: T[]) => T[] | PromiseLike<T[]>;
   /**
+   * A callback for what should happen when a user clicks the input.
+   */
+  onInputFocus?: (selectedItems?: T[]) => T[] | PromiseLike<T[]>;
```

Comment:
Itt this should be onEmptyInputFocus or something like that, because that's the only time it's used. People might get confused.